breadcord/helpers.py

```diff
@@ -122,7 +122,7 @@ def simple_button(
     emoji: str | discord.Emoji | discord.PartialEmoji | None = None,
     row: int | None = None,
 ):
-    """A substitute for ``discord.ui.button`` which generates a custom ID for you.
+    """Substitute for ``discord.ui.button`` which generates a custom ID for you.
```

Comment:
See previous comment @ L27.

```suggestion
    """A substitute for ``discord.ui.button`` which generates a custom ID for you.
```